lib/rsk-utils-legacy.js

```diff
@@ -26,7 +26,7 @@ var waitForSync = (rskClients) => {
   });
 };
 
-var waitForBlock = (rskClient, blockNumber, waitTime = 200, maxAttempts = 600) => {
+var waitForBlock = (rskClient, blockNumber, waitTime = 600, maxAttempts = 1000) => {
```

Comment:
@jeremy-then, I had to increase it in some tests, take a look if I haven't increased too much. You said that for some other tests, these values couldn't be that high. 🙏

Reply:
If the tests are running now for the environment these tests are running, that's fine.
Let's consider putting these values in the configuration, like in the `.env` file. What do you think? Could this be done in the scope of this pr? If not, let's create a ticket to handle that later.

Reply:
Addressed, can you take a look @jeremy-then?

Reply:
Looking good. The names look a bit general, what about specifying them a little more? Like:

```
waitForBlockAttemptTimeMillis = process.env.WAIT_FOR_BLOCK_ATTEMPT_TIME_MILLIS || 600
```

```
waitForBlockMaxAttempts = process.env. WAIT_FOR_BLOCK_MAX_ATTEMPTS || 1000;
```

And let's reduce the defualt values to something like `200` for the waiting and `160` for the attempts, since in a local computer having such a high value won't do any good, but slow down the tests.

Also, we can add a brief description comment above these new env variables in the `.env-example` file saying why we use them and need to update them depending on the environment.

```
# Amount of time in milliseconds for the `waitForBlock` utility function to wait. Configurable because depending on the resources of the machine the tests are running on, the wait time might change. In a machine with little resources (CPU, RAM, disk), a small wait time might not be enough because blocks can be mined slow and  `waitForBlock` might fail with a message like `Block number 800 never reached, last seen was 600`, or `Blockchain not advancing after attempting to find a new block 80 times checking every 200 milliseconds. Couldn't reach block number 800. Last block number seen was: 600`. In a machine with enough resources having a high wait time might be a waste of time since the tests would run slower because if this wait time. In this case, it can be set to a small value. `200` recommended for most machines with enough resources. `500`, `600`, etc., or more for machine with limited resources. Adjust as needed, starting with low values so the tests run as fast as they can.
WAIT_FOR_BLOCK_ATTEMPT_TIME_MILLIS=

# Max attempts for the `waitForBlock` utility function to 'wait' for the given block, trying to find that block once every `WAIT_FOR_BLOCK_ATTEMPT_TIME_MILLIS`. The same as the `WAIT_FOR_BLOCK_ATTEMPT_TIME_MILLIS` variable, the value for this variable could be updated depending on the machine the tests are running on. `80` recommended for most machines with enough resources. `160`, `250` or more for machine with limited resources. Adjust as needed, starting with low values so the tests run as fast as they can.
WAIT_FOR_BLOCK_MAX_ATTEMPTS=
```

If the descriptions are too long, we could move them to the `README.md` file. But I think it's important to be as detailed as possible, so we don't have other developers guessing why the test failed with that message.

Sorry that I was not clearer when I proposed the changes.

Reply:
Followed and applied the suggestions.